CareKitStore/CareKitStore/StoreCoordinator/OCKStoreCoordinator.swift

```diff
@@ -208,7 +208,7 @@ open class OCKStoreCoordinator: OCKAnyStoreProtocol {
         // a single combined result is sent through the stream.
 
         // Create a stream that outputs a single empty result
-        let initialResults: AsyncLazySequence<[[T]]> = [[]].async
+        let initialResults: AsyncSyncSequence<[[T]]> = [[]].async
```

Comment:
All of these are related to 0.1.0 changes in https://github.com/apple/swift-async-algorithms/pull/246